lib/sonata/channelWeeklyAirdrop/processChannelReward.ts

```diff
@@ -0,0 +1,20 @@
+import extractAddresses from "@/lib/privy/extractAddresses";
+import pregenerateChannelWallet from "@/lib/privy/pregenerateChannelWallet";
+import { stack } from "@/lib/stack/client";
+import { eventAirdropChannel } from "@/lib/stack/events";
+import { ChannelStats } from "@/types/ChannelStats";
+
+async function processChannelReward({ channelId, addresses }: ChannelStats, points: number) {
+  let account = addresses?.[0];
+  if (!account) {
+    const result = await pregenerateChannelWallet(channelId);
+    const wallet = await result.json();
+    account = extractAddresses(wallet.linked_accounts)[0];
```

Comment:
I feel like this is a common, shared request to convert `channelId` => `account`
- do we have a shared lib you could import here?
- if no, is it worth creating that lib to DRY code?